egs/tedlium/s5/local/confidence_calibration.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+. cmd.sh
+. path.sh
+
+# Global options,
+graph=exp/tri3/graph
+arpa_gz=db/cantab-TEDLIUM/cantab-TEDLIUM-pruned.lm3.gz
+lmwt=13
+
+# Dev-set options,
+dev_data=data/dev
+dev_latdir=exp/tri3_mmi_b0.1/decode_dev_it4
+
+# Eval-set options,
+eval_data=data/test
+eval_latdir=exp/tri3_mmi_b0.1/decode_test_it4
+
+. utils/parse_options.sh
+set -euxo pipefail
+
+# Derived options,
+dev_caldir=$dev_latdir/confidence_$lmwt
+eval_caldir=$eval_latdir/confidence_$lmwt
+
+###### Data preparation,
+# Prepare filtering for excluding data from train-set,
+word_filter=$(mktemp)
+awk '{ keep_the_word = $1 !~ /^(\[.*\]|<.*>|%.*|!.*|-.*|.*-)$/; print $0, keep_the_word }' \
```

Comment:
a comment that this prints out either "1 some-word" for words to keep, or "0 some-word" for words to reject such as <cough>, would be nice.